remoteappmanager/db/interfaces.py

```diff
@@ -0,0 +1,81 @@
+from abc import ABCMeta, abstractmethod
+
+from traitlets import HasTraits, Unicode, Bool
+
+
+class ApplicationConfig(HasTraits):
```

Comment:
If you want a quick function for comparing you could even compare the `json` strings of the two objects. But a simple function comparing the attributes should be fine. They are not many of them and the interface is not going to change so often.